fsw/src/sc_cmds.c

```diff
@@ -773,4 +775,9 @@ void SC_ManageTable(SC_TableType type, int32 ArrayIndex)
         }
     }
 
+    if (Result == CFE_SUCCESS)
+    {
+        CFE_EVS_SendEvent(SC_MANAGE_TABLE_INF_EID, CFE_EVS_EventType_INFORMATION, "Table manage command.");
```

Comment:
@jphickey Is it possible we never sent a success event here because it floods the system whenever running an RTS (CFE TIME sends this command whenever a table needs to be managed)? We also don't increment the command success counter?

Reply:
I don't understand, could you elaborate?  Specifically what's the flow that would cause flooding and what's the relation to CFE TIME?

Reply:
@skliper Oops, not CFE TIME,  CFE TABLE Services. I'm not sure if it will flood the system or not, I haven't tested it. But I assumed that was the reason why we never added an event for this command. Will this get called only when a table is updated externally from SC? In that case, it might not be a problem.

Reply:
@pepepr08 Oh, yeah I see now where SC calls CFE_TBL_NotifyByMessage to register the SC_MANAGE_TABLE_CC with CFE Table services.  I wish it was using a different MID so it's obvious it's not a ground command.  Either way, should only get called when a table is updated using table services (worth testing to confirm though).  Interestingly SC is the only common/shared app I could find that uses the NotifyByMessage API.  Either way since it's really just internal TBL/SC coordination I don't think it falls under the "common ground command handling" need to send an event when it happens.  I'm not aware of any other app that reports table management in an event.

Reply:
Yes correct - no other apps have an event for table handling, and it was only noted in the issue here because SC is the only app that handles tables as part of the processing of a direct command (`SC_ManageTableCmd()`).

So for now I'll remove the event, and add a comment noting that it is an exception.